lib/resque/failure/extra.rb

```diff
@@ -67,6 +88,8 @@ def each(*args,&block)
       def action_by(opts={}, &block)
         action = opts.delete(:action).to_sym || :requeue
         arg = opts.delete(:arg)
+        opts.delete(:older_than) if opts[:older_than].empty?
```

Comment:
We shouldn't treat `older_than` in a special way, drop all empty/nil value keys.